src/actions/secrets.js

```diff
@@ -1,5 +1,7 @@
+import sanityClient from 'part:@sanity/base/client'
 import {defer} from 'rxjs'
-import client from 'part:@sanity/base/client'
+
+const client = sanityClient.withConfig({apiVersion: '2021-05-17'})
```

Comment:
Attempting to QA by installing this in an existing test studio. I think I'm getting an error on this line

```
Uncaught error
_client.default.withConfig is not a function

Stack:

TypeError: 
    at Object.<anonymous> (http://localhost:3333/static/js/app.bundle.js:56301:30)
    at __webpack_require__ (http://localhost:3333/static/js/vendor.bundle.js:713:30)
    at fn (http://localhost:3333/static/js/vendor.bundle.js:118:20)
    at Object.<anonymous> (http://localhost:3333/static/js/app.bundle.js:257205:16)
    at __webpack_require__ (http://localhost:3333/static/js/vendor.bundle.js:713:30)
    at fn (http://localhost:3333/static/js/vendor.bundle.js:118:20)
    at Object.<anonymous> (http://localhost:3333/static/js/app.bundle.js:257126:37)
    at __webpack_require__ (http://localhost:3333/static/js/vendor.bundle.js:713:30)
    at fn (http://localhost:3333/static/js/vendor.bundle.js:118:20)
    at Object.<anonymous> (http://localhost:3333/static/js/app.bundle.js:257068:116)

```

Will these `sanity-plugin-mux-input` changes require a more recent sanityClient? If so we will need to version this as a major change

Reply:
Yes, not specifying a version in more recent versions of the studio is deprecated and will issue warnings. I would suggest that we follow the new standard of versioning the API and bump the version to 1.0.0.

This comment made me realize that not all client instances had been versioned, which is now fixed in f76076d

See: https://www.sanity.io/help/studio-client-specify-api-version and https://www.sanity.io/docs/api-versioning